packages/app/src/components/PageEditor/EmojiPicker.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC, useRef, useEffect } from 'react';
+import 'emoji-mart/css/emoji-mart.css';
```

Comment:
- please import the css in both `_vendor.scss` and `_apply-colors.scss`  not in this file
- you don't need to set any styles for emojimart on this PR